utilities/plotting/cornerPlot.m

```diff
@@ -6,7 +6,7 @@ function cornerPlot(results, options)
     % --------
     % >>> cornerPlot(result, 'smooth', false);
     % >>> cornerPlot(result, 'smooth', false, 'params', [1, 3]);  % should plot 1st and 3rd fitted parameters only. 
-    % >>> cornerPlot(result, 'smooth', true, 'smoothingFactor', 0.5);
+    % >>> cornerPlot(result, 'smooth', true);
```

Comment:
I think you can remove this snippet, it was added to illustrate `smoothingFactor`